packages/code/src/Code.tsx

```diff
@@ -178,13 +183,6 @@ interface CodeProps extends Omit<SyntaxProps, 'onCopy'> {
    */
   chromeTitle?: string;
 
-  /**
-   * When true, whitespace and line breaks will be preserved.
-   *
-   * default: `true`
-   * */
-  multiline?: boolean;
-
   /**
    * When true, allows the code block to be copied to the user's clipboard
```

Comment:
This description could be better: `If present, displays copy bar next to Code component. The function pased here is invoked when copy button is clicked`. Additionally, noticed that it's not possible to both `showWindowChrome` and `onCopy`. We should probably indicate this through the typing as well

Reply:
Not trying to overhaul too many existing issues in this component, but I'll update the description.

`showWindowChrome` and `onCopy` are compatible, even if the window chrome and onCopy won't be rendered at the same time. In some edge cases, this will make implementation easier, so I'm not sure I'd want to prevent this.